### PR TITLE
[PD-7051] filter chips use meta filters

### PR DESCRIPTION
**Pull request by nate998877**
## What does this PR do?
Changes the filter chips to use the active filters from meta.

## Notes to Reviewer
The meta filters are slugs vs query strings. Until the new-URL format is added these will be broken.

### CODE REVIEW

**Review by jeremytubbs — CHANGES_REQUESTED** [review]
Filter chips should only be links.. the method hitEnter and the eventListeners should longer be required.

**Comment by nate998877** [comment]
> Filter chips should only be links.. the method hitEnter and the eventListeners should longer be required.

I thought those might be for navigation purposes, so intentionally left them in, since I don't know.

File: src/components/Search/AppSearchFilterChip.vue [hunk 1]
```diff
@@ -9,76 +9,17 @@
 </template>
 
 <script>
-import buildUrl from "axios/lib/helpers/buildURL"
 const ENTER_KEY = 13
```
Comment (jeremytubbs) [inline src/components/Search/AppSearchFilterChip.vue:12]: const no longer needed

File: src/components/Search/AppSearchFilterChip.vue [hunk 2]
```diff
@@ -9,76 +9,17 @@
...
-            query = {...query, ...input}
-
-            if (!Array.isArray(names)) {
-                names = [names]
-            }
-            for (let item of names) {
-                delete query[item]
-            }
-            return buildUrl(this.path, query)
+            required: true,
```
Comment (jeremytubbs) [inline src/components/Search/AppSearchFilterChip.vue:22]: should be `required: false`

**Review by jeremytubbs — CHANGES_REQUESTED** [review]
Minor changes

**Review by jeremytubbs — APPROVED** [review]

**Pull request merged**